common/utils/src/main/resources/error/error-classes.json

```diff
@@ -6310,5 +6330,11 @@
     "message" : [
       "Failed to get block <blockId>, which is not a shuffle block"
     ]
+  },
+  "_LEGACY_ERROR_USER_RAISED_EXCEPTION" : {
```

Comment:
this is a new prefix. how is it different from `_LEGACY_ERROR_TEMP`?

Reply:
It's not temporary.... But I still want it to be filtered out.